### DISCUSSION TITLE
Unable to select multiple Dependabot alerts to dismiss?

### DISCUSSION THREAD

**Discussion by jrimmer-bendhealth** [community/community#135809] [Code Security] [answered]
### Select Topic Area

Question

### Body

Documentation (https://docs.github.com/en/code-security/dependabot/dependabot-alerts/viewing-and-updating-dependabot-alerts -- "You can sort the list of alerts, and you can click into specific alerts for more details. You can also dismiss or reopen alerts, either one by one or by selecting multiple alerts at once.") says we can select multiple Dependabot alerts to reopen or dismiss; however, the checkboxes to do so do not appear, and I can find no documentation showing WHY they aren't appearing.

What am I missing?

**Comment by jrimmer-bendhealth** [comment, marked as answer]
Ah, I found it. If you're looking at your entire organization's view of Dependabots, the checkboxes don't appear, not even if you filter by a single repository.

If you go into the *Repository's* view, then you get the checkboxes.

**Comment by makskustomer** [comment]
Its honestly disappointing that there isnt a whole org view to dismiss; there are situations where packages are not exploitable or certain cve's especially in a certain scope such as development.
